src/browser/api/system.js

```diff
@@ -610,6 +610,38 @@ exports.System = {
             }
         });
     },
+    getCookies: function(opts, callback, errorCallback) {
+        const { url, name } = opts;
+        if (url && url.length > 0 && name && name.length > 0) {
+            session.defaultSession.cookies.get({ url, name }, (error, cookies) => {
+                if (error) {
+                    log.writeToLog(1, error, true);
```

Comment:
Why don't we forward the error? `errorCallback(error)`

Reply:
Updated as requested